### PR TITLE
Migrate go mod

### PR DESCRIPTION
**Pull request by flapezoti**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
-WORKDIR /go/src/github.com/topfreegames/mqtt-history
-RUN dep ensure
-RUN go install github.com/topfreegames/mqtt-history
+# Build a static binary.
+RUN CGO_ENABLED=0 GOOS=linux go build -mod vendor -a -installsuffix cgo -o mqtt-history .
+
+# Verify if the binary is truly static.
+RUN ldd /src/mqtt-history 2>&1 | grep -q 'Not a valid dynamic program'
+
+FROM alpine
```
Comment (martinsrso) [inline Dockerfile]: add a fixed version of the alpine.

File: Dockerfile [hunk 2]
```diff
@@ -1,16 +1,23 @@
-FROM golang:1.8.3-alpine
+FROM golang:1.14-alpine AS build
```
Comment (martinsrso) [inline Dockerfile]: WDYT, use 1.15 go version?

Comment (flapezoti) [reply Dockerfile]: Agreed, updated to 1.15 everywhere

File: Dockerfile (hunk 1 above)
Comment (flapezoti) [reply Dockerfile]: Added it, thanks

**Pull request merged**